src/tests/Microsoft.PowerFx.Interpreter.Tests/SetFunctionTests.cs

```diff
@@ -385,6 +385,22 @@ public static ReadOnlySymbolTable NewMutableFromRecord(RecordType type)
         {
             var symTable = ReadOnlySymbolTable.NewFromRecord(type, allowMutable: true);
             return symTable;
+        }
+
+        [Fact]
+        public void NewErrorMessageWithTypes()
```

Comment:
>NewErrorMessageWithTypes [](http://example.com/codeflow?start=20&length=24)

Add another test case exploring a different locale. For that you can provide a CultureInfo to PowerFxConfig object.
`var powerFxConfig = new PowerFxConfig(CultureInfo.CreateSpecificCulture("fr-FR"));`